samples/mongo-sample/tye.yaml

```diff
@@ -0,0 +1,21 @@
+services:
+- name: mongo
+  image: mongo
+  env:
+    - name: ME_CONFIG_MONGODB_ADMINUSERNAME
+      value: root
+    - name: ME_CONFIG_MONGODB_ADMINPASSWORD
+      value: example
+  bindings:
+    - port: 27017
+- name: mongo-express
+  image: mongo-express
```

Comment:
The mongo-express container image points to the host name mongo by default https://github.com/mongo-express/mongo-express#usage-docker